Tests/Spec/tentacle_scenario_08_tentacle_comms_port_spec.rb

```diff
@@ -35,6 +35,7 @@
   it { should have_endpoint("https://#{ENV['COMPUTERNAME']}:10900/") }
 end
 
+=begin
```

Comment:
Dont think this should be commented out. It should use the new "check the folder" approach.